napari_btrack/widgets/_general.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+import magicgui
+import napari
+
+
+def create_input_widgets():
+    """Create widgets for selecting labels layer and TrackerConfig"""
+
+    tooltip = (
+        "Select a 'Labels' layer to use for tracking.\n"
+        "To use an 'Image' layer, first convert 'Labels' by right-clicking "
+        "on it in the layers list, and clicking on 'Convert to Labels'"
+    )
+    segmentation_selector = magicgui.widgets.create_widget(
+        annotation=napari.layers.Labels,
+        name="segmentation_selector",
+        label="segmentation",
+        options={"tooltip": tooltip},
+    )
+
+    tooltip = "Select a loaded configuration.\nNote, this will update values set below."
+    config_selector = magicgui.widgets.create_widget(
+        value="cell",
+        name="config_selector",
+        label="base config",
+        widget_type="ComboBox",
+        options={
+            "choices": ["cell", "particle"],
+            "tooltip": tooltip,
+        },
+    )
```

Comment:
I just thought it would be more readable than having multiline strings defined inside the `options` dictionaries. Plus they would be indented twice more, so the strings would be split across even more lines. But yeah naming them e.g. `segmentation_selector_tooltip` makes sense, and helps ensure we're always using the correct tooltip for each widget

Reply:
I've renamed all the 'tooltip' variables to be specific to each widget